docs/security.html

```diff
@@ -416,7 +448,10 @@
                 the user for client connections. In this example, clients connect to the broker as user <i>alice</i>.
             </li>
             <li>Pass the JAAS config file location as JVM parameter to each client JVM:
-                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
+                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre>
```

Comment:
I understand the reasoning, but I think we need to do a better job at guiding users towards what we consider the better option (and mention the advantage of not being restricted to one authentication user for all clients). I still think we could do what I suggested and then have a link to the static configuration with a blurb stating that it's similar to how it's done for the broker.

What do you think?